### PR TITLE
remove r-nightly-packages from build table

### PR DESCRIPTION
**Pull request by boshek**
Removed `r-nightly-packages` from build table. I have kept it in the Error Logs because that history could conceivably be useful.

### CODE REVIEW

**Comment by boshek** [comment]
@assignUser - just pinging this for a review

File: crossbow-nightly-report.Rmd [hunk 1]
```diff
@@ -184,7 +184,8 @@ nightly %>%
 
 ```{r results="asis"}
 map_params <- nightly %>%
-  distinct(task_name, build_type)
+  distinct(task_name, build_type) %>% 
+  filter(!task_name %in% c("r-nightly-packages")) # tasks that are no with us
```
Comment (assignUser) [inline crossbow-nightly-report.Rmd:188]:
```suggestion
  filter(!task_name %in% c("r-nightly-packages", "test-r-rstudio-r-base-4.2-opensuse42")) # tasks that are no longer active
```

**Review by assignUser — CHANGES_REQUESTED** [review]
Nice, thanks! I add the also defunct opensuse42 job.

**Review by assignUser — APPROVED** [review]
Thanks!

**Pull request merged**